fastapi/applications.py

```diff
@@ -190,6 +196,24 @@ async def redoc_html(req: Request) -> HTMLResponse:
 
             self.add_route(self.redoc_url, redoc_html, include_in_schema=False)
 
+        if self.liveness_probe_url:
+
+            async def liveness(req: Request) -> JSONResponse:
+                if not self.probes_state.alive:
+                    return JSONResponse({"status": "DOWN"}, status_code=503)
+                return JSONResponse({"status": "OK"})
+
+            self.add_route(self.liveness_probe_url, liveness, include_in_schema=False)
+
+        if self.readiness_probe_url:
+
+            async def readiness(req: Request) -> JSONResponse:
```

Comment:
As an example, I have the following case: 
_the readiness of the endpoint is polled constantly, not only until the service is ready to receive traffic._

So this handler contains logic to check the readiness of the module to receive traffic.
Probably it goes against the rules, but this is a life case and ops engineers say that this is a frequent case.
Perhaps it is worth giving the opportunity to redefine the readiness method.

What do u think?